tests/integration_tests/test_matchmaker.py

```diff
@@ -368,13 +374,20 @@ async def test_matchmaker_info_message(lobby_server, mocker):
         assert "num_players" in queue
 
         assert queue["queue_pop_time"] == "2019-07-01T16:53:21+00:00"
+        assert queue["queue_pop_time_delta"] == math.ceil(
+            config.QUEUE_POP_TIME_MAX / 2
```

Comment:
Why is it half here? I guess it's because it's getting the first queue pop? I vaguely remember scheduling the first pop for half the max time.

Reply:
That's right, this is the first queue pop, so I wanted to protect the test in case `QUEUE_POP_TIME_MAX` ever changed. [This is where it's used](https://github.com/FAForever/server/blob/develop/server/matchmaker/pop_timer.py#L36) in the `PopTimer`.

Reply:
Using the config is fine, I was just trying to remember why the `/ 2` was needed.